R/sensitivity-analysis-results.R

```diff
@@ -0,0 +1,27 @@
+
+#' @title SensitivityAnalysisResults
+#' @docType class
+#' @description  Results of a sensitivity analysis run (either individual or population simulation)
+#' @section Methods:
+#' \describe{
+#'   \item{allPKParameterSensitivitiesFor(pkParameterName, outputPath, totalSensitivityThreshold)}
+#'   {Returns the PKParameterSensitivity for a given pkParameter and output participating to a total sensitivity greater or equal to totalSensitivityThreshold}
+#'   }
+#' @format NULL
+SensitivityAnalysisResults <- R6::R6Class("SensitivityAnalysisResults",
+  inherit = DotNetWrapper,
+  public = list(
+    allPKParameterSensitivitiesFor = function(pkParameterName, outputPath, totalSensitivityThreshold) {

```

Comment:
yep. And this could be defined in the config object for SA
This value is crucial so that you can get all parameters that account for xxx of the total sensitivity. For example knowing what parameters participate to 90% of the overall model sensitivity. That way, you only focus on the parameters that really matter

Reply:
done